data/human/culture conversations.txt

```diff
@@ -971,6 +971,99 @@ mission "Exotic Life: Glaze"
 			`	You walk back to your ship from the binoculars.`
 				decline
 
+mission "Exotic Life - Bluestone"
+	minor
+	source Bluestone
+	to offer
+		random < 10
+	on offer
+		conversation
+			`You step off your ship and wander around Bluestone's quiet yet functional spaceport. The buildings, made mostly of durable plastic resin, form a quaint but near-uniform red-brown landscape, except for the starkly contrasting cheerful small yellow pub in the center. The spaceport is quiet, but the low hum of activity from a nearby oil drill gives it some life. As you are entering the pub for a drink, you spot an announcement on a bulletin board:`
```

Comment:
I like the descriptions here, but there's maybe too many adjectives in a row ("contrasting cheerful small yellow"). You also seem to be assuming that the player drinks. Maybe something more like this would work better?

```suggestion
			`You step off your ship and wander around Bluestone's quiet yet functional spaceport. The buildings, made mostly of durable plastic resin, form a quaint but near-uniform red-brown landscape, in sharp contrast to the small yellow pub in the center. The spaceport is quiet, but the low hum of activity from a nearby oil drill gives it some life. As you are entering the pub, you spot an announcement on a bulletin board:`
```

Reply:
Great point, I think that does flow quite a bit better.